### ISSUE TITLE
Any suggestion about fine-tuning on another outdoor dataset

### ISSUE THREAD

**Issue by Gaozihui** [isl-org/ZoeDepth#27]
Thanks for your brilliant work.

I am trying to use the pre-trained model to fine-tune on another outdoor dataset. Should I use the pretrained model ZoeD_M12_K.pt or the pretrained model without fine-tuning on any dataset (which is not released I think) ? I have tried the former one but I don't see an obvious droping on the training.
![image](https://user-images.githubusercontent.com/58518515/234504331-c6c26a3b-9eec-41aa-8302-77c4081ea6aa.png)

Thank you in advance for any advice you can offer.

**Comment by MACILLAS** [comment]
Hi @Gaozihui I am also trying to fine-tune on another outdoor dataset. I'm getting stuck thou when I run the train_mono.py with ZoeD_MK12_K.pt I get a bunch of size mismatch errors. I'm wondering if you had this issue and if you had any pointers?

Thanks in advance! 

```
python train_mono.py -m zoedepth -d kitti --pretrained_resource="url::https://github.com/isl-org/ZoeDepth/releases/download/v1.0/ZoeD_M12_K.pt"
```

**Comment by Gaozihui** [comment]
> Hi @Gaozihui I am also trying to fine-tune on another outdoor dataset. I'm getting stuck thou when I run the train_mono.py with ZoeD_MK12_K.pt I get a bunch of size mismatch errors. I'm wondering if you had this issue and if you had any pointers?
> 
> Thanks in advance!
> 
> ```
> python train_mono.py -m zoedepth -d kitti --pretrained_resource="url::https://github.com/isl-org/ZoeDepth/releases/download/v1.0/ZoeD_M12_K.pt"
> ```

Have you solved this problem?  
```
python train_mono.py -m zoedepth -d nuscenes --batch_size 8 \
 --trainer zoedepth --pretrained_resource "local::/mnt/nas/share/home/gzh/ZoeDepth/ZoeD_M12_K.pt" config_version kitti
```
I think maybe you leave out the 'config_version' term. Please try it.

**Comment by ariqshadi** [comment]
Hi, im just wondering, when you fine tune using a custom dataset, are the label depth images for the training in a relative depth value or in a metric depth value?

**Comment by thejerrycheng** [comment]
I have the same problem here, should the custom dataset be metric depth or in relative depth

**Comment by MACILLAS** [comment]
I think the main idea is that the decoder learns metric depth, so you fine-tune with metric depth dataset. If you have a relative depth dataset I think you can just evaluate MiDASv3 on it directly?
